docs/source/about.rst

```diff
@@ -1,3 +1,5 @@
+|RAT banner|
```

Comment:
The _docs/examples_ can be moved to _examples/manual_

Reply:
Okay. Moved.